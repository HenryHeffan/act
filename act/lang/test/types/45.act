
defproc testme (chan!(int) X)
{
  int x, y;
  chp {
    X!y ; X?x
  }
}

testme x;
