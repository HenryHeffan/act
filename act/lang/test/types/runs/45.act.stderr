In expanding testme (45.act:2)
In expanding ::<Global>
Receive operation on an output channel.
	Channel: X
